src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +15,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+
+        return (user != null && user.getPassword().equals(password)) ? true : false;
```

Comment:
really not bad try, but... seriously?))
```suggestion
        return user != null && user.getPassword().equals(password));
```